src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -68,7 +68,11 @@
             {% set render_preferred_choices = true %}
             {{- block('choice_widget_options') -}}
             {%- if choices|length > 0 and separator is not none -%}
-                <option disabled="disabled">{{ separator }}</option>
+                {%- if separator_html is defined and separator_html is same as(false) -%}
+                    <option disabled="disabled">{{ separator }}</option>
```

Comment:
we need to make a similar change in `foundation_5_layout.html.twig` too

Reply:
good point, I've made the necessary changes